app/src/main/java/org/fossasia/susi/ai/adapters/viewholders/ChatViewHolder.java

```diff
@@ -55,4 +69,78 @@ public ChatViewHolder(View view, ClickListener clickListener ,int myMessage) {
             default:
         }
     }
+
+    /**
+     * Inflate ChatView
+     *
+     * @param model the ChatMessage object
+     * @param viewType the viewType
+     * @param flag the flag
+     * @param query the query
+     */
+    public void setView(ChatMessage model, int viewType, boolean flag, String query) {
+        if (model != null) {
+            try {
+                switch (viewType) {
+                    case USER_MESSAGE:
+                        chatTextView.setText(model.getContent());
+                        timeStamp.setText(model.getTimeStamp());
+                        if (model.getIsDelivered())
+                            receivedTick.setImageResource(R.drawable.check);
+                        else
+                            receivedTick.setImageResource(R.drawable.clock);
+
+                        chatTextView.setTag(this);
+                        if (flag) {
+                            String text = chatTextView.getText().toString();
+                            SpannableString modify = new SpannableString(text);
+                            Pattern pattern = Pattern.compile(query, Pattern.CASE_INSENSITIVE);
+                            Matcher matcher = pattern.matcher(modify);
+                            while (matcher.find()) {
+                                int startIndex = matcher.start();
+                                int endIndex = matcher.end();
+                                modify.setSpan(new BackgroundColorSpan(Color.parseColor("#ffff00")), startIndex, endIndex, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
+                            }
+                            chatTextView.setText(modify);
+
+                        }
+                        timeStamp.setTag(this);
+                        receivedTick.setTag(this);
+                        break;
+                    case SUSI_MESSAGE:
+                        Spanned answerText;
```

Comment:
Add this here 
```
if(model.getActionType().equals(Constant.ANCHOR)) {
    chatTextView.setLinksClickable(true);
    chatTextView.setMovementMethod(LinkMovementMethod.getInstance());
} else {
    chatTextView.setLinksClickable(false);
}
```
And delete 
```
android:clickable="true"
android:linksClickable="true"
android:longClickable="true"
```
from [here](https://github.com/fossasia/susi_android/blob/development/app/src/main/res/layout/item_susi_message.xml#L35)